internal/provider/kubernetes/controller.go

```diff
@@ -1049,6 +1077,20 @@ func (r *gatewayAPIReconciler) watchResources(ctx context.Context, mgr manager.M
 		return err
 	}
 
+	// Watch BackendTLSPolicy
```

Comment:
also need to include predicates
https://github.com/envoyproxy/gateway/blob/cf46fbe776918ad19444e26d637ffcc79676ca23/internal/provider/kubernetes/predicates.go#L476

&&

https://github.com/envoyproxy/gateway/blob/cf46fbe776918ad19444e26d637ffcc79676ca23/internal/provider/kubernetes/predicates.go#L151

&&

Reply:
backendTLSPolicy does not use secret

Reply:
sure `Core` support is for `ConfigMap` as stated in https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.BackendTLSPolicyConfig
feel free to only implement for `ConfigMap` in this PR, but we'll need to also support `Secret` CACertRefs in the future